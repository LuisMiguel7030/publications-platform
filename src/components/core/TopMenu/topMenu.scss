.top-menu {

    background-color: #ffff;
    border-bottom: 1.5px rgba(134, 134, 134, 0.432) solid;
    padding: 16px;
    box-shadow: 0px 3px 15px 0px rgba(0,0,0,0.09);

    &__title {
        font-size: 16px;
    }
}