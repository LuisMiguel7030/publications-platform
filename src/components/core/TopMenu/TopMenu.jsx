import './topMenu.scss'
import PropTypes from 'prop-types';
import React from 'react';

function TopMenu({ title }) {
    return (
        <header className="top-menu" >
            <h1 className="top-menu__title">{title}</h1>
        </header>
    )
}

TopMenu.propTypes = {
    title: PropTypes.string
};

TopMenu.defaultProps = {
    title: null
};

export default TopMenu;