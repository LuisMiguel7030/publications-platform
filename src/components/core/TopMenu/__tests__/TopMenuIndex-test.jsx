import React from 'react';
import { mount } from 'enzyme';
import 'jest';
import TopMenu from '../TopMenu';

describe('Sign in page form', () => {
    let result;

    it('renders correctly', () => {
        result = mount(<TopMenu />)

        expect(result).toMatchSnapshot();
    });

    it('should render with the correct porp', () => {
        result = mount(<TopMenu title="TEST"/>)

        expect(result.prop('title')).toBe('TEST');
    });
});