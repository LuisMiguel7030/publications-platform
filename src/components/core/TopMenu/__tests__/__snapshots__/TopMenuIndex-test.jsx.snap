// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign in page form renders correctly 1`] = `
<TopMenu
  title={null}
>
  <header
    className="top-menu"
  >
    <h1
      className="top-menu__title"
    />
  </header>
</TopMenu>
`;
